build.gradle.kts

```
@@ -0,0 +1,67 @@
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+
+plugins {
+    id("java-gradle-plugin")
+    id("com.gradle.plugin-publish") version "0.12.0"
+    groovy //TODO remove it at the end of the migration
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+repositories {
+    mavenCentral()
+    gradlePluginPortal()
+}
+
+dependencies {
+    implementation("io.gatling:gatling-enterprise-plugin-commons:1.5.3") {
+        exclude("com.fasterxml.jackson.core")
+    }
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.14.2")
+    implementation("org.apache.ant:ant:1.10.11")
+
+    testImplementation("org.spockframework:spock-core:1.3-groovy-2.5") {
+        exclude("groovy-all")
+    }
+    testImplementation("com.github.stefanbirkner:system-rules:1.19.0") {
+        exclude("junit-dep")
+    }
+    testImplementation("junit:junit:4.13.2")
+    testImplementation("commons-io:commons-io:2.11.0")
+}
+
+tasks{
+    test {
+        testLogging {
+            events = setOf(TestLogEvent.PASSED, TestLogEvent.SKIPPED, TestLogEvent.FAILED)
+            exceptionFormat = TestExceptionFormat.FULL
+            showStandardStreams = false
+        }
+        val procAvailable = Runtime.getRuntime().availableProcessors()
+        maxParallelForks = if (procAvailable.div(2) == 0 ) 1 else procAvailable.minus(1)
```

What is that?
Why a division by 2 only to check if it's less than 2?

From [gradle documentation](https://docs.gradle.org/current/userguide/performance.html#execute_tests_in_parallel):
```suggestion
        maxParallelForks = (Runtime.getRuntime().availableProcessors() / 2).takeIf { it > 0 } ?: 1
```

---

`procAvailable.minus(1)`was a test on my laptop cause with `procAvailable.div(2)` my laptop cpu  goes to sleep half the time ;-)